test/test_mechanism_algorithms.jl

```diff
@@ -110,6 +110,20 @@ end
             configuration_derivative_to_velocity!(vjoint_from_q̇, joint, qjoint, q̇joint)
             @test isapprox(vjoint, vjoint_from_q̇; atol = 1e-12)
         end
+
+        V_q = RigidBodyDynamics.configuration_derivative_to_velocity_jacobian(x)
+        Q_v = RigidBodyDynamics.velocity_to_configuration_derivative_jacobian(x)
+        RigidBodyDynamics.configuration_derivative_to_velocity_jacobian!(V_q, x)
+        RigidBodyDynamics.velocity_to_configuration_derivative_jacobian!(Q_v, x)
+        for i in 1:100
+            rand!(x)
+            allocs = @allocated RigidBodyDynamics.configuration_derivative_to_velocity_jacobian!(V_q, x)
+            @test allocs == 0
```

Comment:
I'm surprised this passes. Shouldn't it allocate for `i == 1` due to compilation?